plugins/modules/sorcery.py

```diff
@@ -20,8 +20,6 @@
 notes:
     - When all three components are selected, the update goes by the sequence --
       Sorcery -> Grimoire(s) -> Spell(s); you cannot override it.
-    - grimoire handling (i.e. add/remove, including SCM/rsync versions) is not
-      yet supported.
```

Comment:
How about replacing this with `Grimoire handling is supported since community.general 7.3.0.`?